sqlitecluster/SQLiteNode.cpp

```diff
@@ -1819,18 +1866,26 @@ void SQLiteNode::_sendToPeer(SQLitePeer* peer, const SData& message) {
     // We can treat this whole function as atomic and thread-safe as it sends data to a peer with it's own atomic
     // `sendMessage` and the peer itself (assuming it's something from _peerList, which, if not, don't do that) is
     // const and will exist without changing until destruction.
-    peer->sendMessage(_addPeerHeaders(message).serialize());
+    if (peer->forked) {
```

Comment:
Do we use this to send PING/PONG?  I think we want to let those through?

Reply:
We specifically call `peer->sendMessage`directly for PING/PONG for this reason. Everywhere else we call `_sendToPeer`.